motion/core_ext/string/filters.rb

```diff
@@ -1,9 +1,9 @@
 class String
-  # Returns the string, first removing all whitespace on both ends of
-  # the string, and then changing remaining consecutive whitespace
-  # groups into one space each.
+  # Returns the string, first removing all whitespace on both ends of the
+  # string, and then changing remaining consecutive whitespace groups into one
+  # space each.
   #
-  # Note that it handles both ASCII and Unicode whitespace like mongolian vowel separator (U+180E).
```

Comment:
This is hard to answer in a meaningful way, given that I didn't even know they were a thing prior to this commit, let alone what's changed with them in recent history. What I can say is that when `squish`ed, they were being encoded as this:

<img width="907" alt="screenshot 2015-11-14 21 53 08" src="https://cloud.githubusercontent.com/assets/723784/11167118/18708e98-8b1a-11e5-8d60-170d257e8f56.png">

Running the same code in a `rails` console (with ActiveSupport loaded, obviously) results in the following:

<img width="861" alt="screenshot 2015-11-14 21 54 08" src="https://cloud.githubusercontent.com/assets/723784/11167125/3f179028-8b1a-11e5-8cf3-78623a3317df.png">

The responses are obviously different (which may be its own failing spec), but they're also clear in that they aren't actually removing the characters from the beginning/end of the lines as the spec would indicate.